Purchases/Public/RCEntitlementInfo.m

```diff
@@ -43,10 +44,25 @@ - (instancetype)initWithEntitlementId:(NSString *)entitlementId entitlementData:
                                                      store:self.store
                                      unsubscribeDetectedAt:self.unsubscribeDetectedAt
                                     billingIssueDetectedAt:self.billingIssueDetectedAt];
+        self.ownershipType = [self parseOwnershipType:productData[@"ownership_type"]];
+
     }
     return self;
 }
 
+- (RCPurchaseOwnershipType)parseOwnershipType:(NSString * _Nullable)ownershipType {
+    if (!ownershipType) {
+        return RCPurchaseOwnershipTypePurchased;
+    }
+    if ([ownershipType isEqualToString:@"PURCHASED"]) {
+        return RCPurchaseOwnershipTypePurchased;
+    } else if ([ownershipType isEqualToString:@"FAMILY_SHARED"]) {
+            return RCPurchaseOwnershipTypeFamilyShared;
+    } else {
+        return RCPurchaseOwnershipTypeUnknown;
+    }
+}
```

Comment:
@Carlosedo just wanted a sanity check that this is exactly what we'd expect from the backend. this is reading from inside each item in `"subscriptions"`.

Reply:
Also, two things to note here: 
- If the key is entirely missing, I'm assuming that ownershipType = "purchased". This makes the field work in sandbox, and seems like a reasonable default since it was the behavior for everything prior to family sharing. 
- if the key's value isn't `PURCHASED` or `FAMILY_SHARED`, I'm defaulting to unknown. This is so that if we ever want to add a new key, old versions of the SDK would read the new key as `UNKNOWN`, giving us some extra flexibility vs defaulting to `PURCHASED`, which may not be intended.